samples/csharp_dotnetcore/06.using-cards/Cards.cs

```diff
@@ -81,7 +81,7 @@ public static ReceiptCard GetReceiptCard()
                         ActionTypes.OpenUrl,
                         "More information",
                         "https://account.windowsazure.com/content/6.10.1.38-.8225.160809-1618/aux-pre/images/offer-icon-freetrial.png",
-                        "https://azure.microsoft.com/en-us/pricing/"),
+                        value: "https://azure.microsoft.com/en-us/pricing/"),
```

Comment:
why?

Reply:
OK nice catch

Reply:
Sorry for the confusion. There was a bug (that I did not happen to screenshot) where the receipt card wouldn't get displayed because the OpenUrl action had no value argument.